.buildkite/pipeline.elastic-agent-package.yml

```diff
@@ -82,7 +84,7 @@ steps:
           imagePrefix: "core-ubuntu-2204-aarch64"
         env:
           PLATFORMS: "linux/arm64"
-          PACKAGES: "docker"
+          PACKAGES: "docker tar.gz deb rpm"
```

Comment:
My understanding from @pchila is that PACKAGES **should be** command separated, see this comment: https://github.com/elastic/elastic-agent/pull/7203#discussion_r1983000529

@pchila should we switch to comma separated for `PACKAGES`, or are we good after all for `PACKAGES` too?

Reply:
PACKAGES need to be comma-separated, see https://github.com/elastic/elastic-agent/blob/b20efc1801b1b18d9e0b035cea34ff1b942a9066/dev-tools/mage/crossbuild.go#L46-L56

Reply:
Just in case, replaced with `,`